README.md

```diff
@@ -43,6 +43,14 @@ Besides functionality to perform a typical recommend-measure loop, BayBE's highl
 
 ## ⚡ Quick Start
 
+> [!IMPORTANT]
+> You are reading the documentation for the branch shown in GitHub's
+> **branch selector** at the top of the page.

```

Comment:
this note will also be shown on the docs landing page, right?

there is no branch selector there, hence aiming to reduce on confusion you're adding one in another place, why?

Reply:
Have to say that I agree with Martin here: This note makes no sense for the rendered version as it seems to imply that we already have multi version docs, which we don't. If at all, there should be a note that says "You are reading the documentation of the latest version of BayBE available on the main branch. This is not necessarily equal to a version installed via pip." Otherwise, people might think that the version of the full documentation and the other examples changes if they somehow change the selected branch.

Reply:
What we could do is already build and upload the old versions of the docs as it is described in the Multi Version PR Draft: This part of the workflow should already run (as mentioned several times, I unfortunately do not have the time at the moment to look at that in detail) and could add something like "Accessing the documentation versions XXX to YYY is available via a corresponding link"